server/session/sessionController.js

```diff
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken');
```

Comment:
Remove this file from the pull request since you are still working on it. And it's separate issue